CODEOWNERS

```diff
@@ -343,6 +344,7 @@ Kconfig*                                  @tejlmand
 /tests/subsys/nrf_profiler/               @pdunaj @MarekPieta
 /tests/subsys/sdfw_services/              @anhmolt @hakonfam @jonathannilsen
 /tests/subsys/zigbee/                     @milewr
+/tests/subsys/suit/                       @tomchy
```

Comment:
Wouldn't we like to extend the code owners list with other developers in the SUIT team (probably @adsz-nordic, @ahasztag perhaps @e-rk if he'd like). I think it wouldn't block anyone, and the team would be informed about external changes if you'd be on vacation